client/RequestBehavior.ts

```diff
@@ -101,6 +108,81 @@ export class PopupRequestBehavior extends RequestBehavior<BehaviorType.POPUP> {
         }
         return popup;
     }
+
+    private appendOverlay(popup: Window): HTMLDivElement {
+        const createElement = document.createElement.bind(document);
+        const createTextNode = document.createTextNode.bind(document);
+        const appendChild = (node: Node, child: Node) => node.appendChild(child);
+
+        // Overlay background
+        const overlay = createElement('div');
+        const overlayStyle = overlay.style;
+        overlayStyle.position = 'absolute';
+        overlayStyle.top = '0';
+        overlayStyle.right = '0';
+        overlayStyle.bottom = '0';
+        overlayStyle.left = '0';
+        overlayStyle.background = 'rgba(31, 35, 72, 0.8)';
+        overlayStyle.display = 'flex';
+        overlayStyle.flexDirection = 'column';
+        overlayStyle.alignItems = 'center';
+        overlayStyle.justifyContent = 'space-between';
+        overlayStyle.cursor = 'pointer';
+        overlayStyle.color = 'white';
+        overlayStyle.textAlign = 'center';
+        overlayStyle.opacity = '0';
+        overlayStyle.transition = 'opacity 0.6s ease';
+        overlay.addEventListener('click', () => popup.focus());
+
+        // Top flex spacer
+        appendChild(overlay, createElement('div'));
+
+        // Explainer text
+        const text = createElement('div');
+        const textStyle = text.style;
+        appendChild(text, createTextNode('A popup is active,'));
+        appendChild(text, createElement('br'));
+        appendChild(text, createTextNode('click anywhere to refocus.'));
+        textStyle.fontSize = '24px';
+        textStyle.fontWeight = '600';
+        textStyle.lineHeight = '40px';
```

Comment:
If Muli is available on the page, I think we should use it. I.e.:
`textStyle.fontFamily = 'Muli', fallbacks...`